resteasy-dependencies-bom/pom.xml

```diff
@@ -791,6 +792,16 @@
                 <artifactId>rxjava</artifactId>
                 <version>${version.io.reactivex.rxjava2-rxjava}</version>
             </dependency>
+            <dependency>
```

Comment:
Shouln'd be the "wrong (old)" jboss-marshaling excluded ?

Reply:
jboss-marshalling cames from org.wildfly.arquillian.wildfly-arquillian-container-managed. This artifact is used just in TS. I exclude marshaling libraries and 20 testcases from spring modules fails with: `java.lang.NoClassDefFoundError: org.jboss.marshalling.ClassResolver`. So I recommend to upgrade marshalling library to the latest version used in WF (like this PR does) and not to exclude it ...